common/src/test/java/org/apache/uniffle/common/util/RssUtilsTest.java

```diff
@@ -177,6 +181,24 @@ public void testGenerateServerToPartitions() {
     assertEquals(serverToPartitions.get(server4), Sets.newHashSet(2, 4));
   }
 
+  @Test
+  public void testGetConfiguredLocalDirs() {
+    RssConf conf = new RssConf();
+    withEnv(RssUtils.RSS_LOCAL_DIR_KEY, "/path/a", () -> {
```

Comment:
There is a better way to test environment variables:

https://github.com/apache/incubator-uniffle/blob/6ddf8a7f0ae7069b869e5734c40226ac2c81494d/server/src/test/java/org/apache/uniffle/server/ShuffleServerConfTest.java#L33-L38

https://github.com/apache/incubator-uniffle/blob/6ddf8a7f0ae7069b869e5734c40226ac2c81494d/server/src/test/java/org/apache/uniffle/server/ShuffleServerConfTest.java#L49-L58

Reply:
Thanks for abstracting all use case of `setEnv()`. However now you are mixing two things in one PR.
I would suggest split them for easier review (and revert, just in case).

Reply:
OK, no big deal.